src/silverlabnwb/nwb_file.py

```diff
@@ -753,6 +757,13 @@ def add_imaging_plane(self, name, manifold, description,
                                      description='Red channel, typically used for reference.',
                                      emission_lambda=float(opto_metadata['emission_lambda']['red']))
             channels.append(channel)
+        if self.compress:
+            wrapped_manifold = H5DataIO(data=manifold,
```

Comment:
Just noticed, since we're using `hdmf` directly now, we should also include it in our `setup.py`.